app/models/models.py

```diff
@@ -39,102 +43,137 @@ def hashed_password(password):
 
     @staticmethod
     def get_user_by_id(user_id):
-        user = User.query.filter_by(id=user_id).first()
+        users = user_container.query_items(
+            query="SELECT * FROM users u WHERE u.id = @userId",
+            parameters=[dict(name="@userId", value=user_id)],
+            enable_cross_partition_query=True,
+        )
+        user = list(users)[0]
+
+        return user
+
+    @staticmethod
+    def get_user_by_email(user_email):
+        users = user_container.query_items(
+            query="SELECT * FROM users u WHERE u.email = @userEmail",
+            parameters=[dict(name="@userEmail", value=user_email)],
+            enable_cross_partition_query=True,
+        )
+        user = list(users)[0]
+
         return user
 
     @staticmethod
     def get_user_with_email_and_password(email, password):
-        user = User.query.filter_by(email=email).first()
-        if user and bcrypt.check_password_hash(user.password, password):
-            return user
+        users = user_container.query_items(
+            query="SELECT * FROM users u WHERE u.email = @userEmail AND u.password = @userPassword",
+            parameters=[
+                dict(name="@userEmail", value=email),
+                dict(name="@userPassword", value=password),
+            ],
+            enable_cross_partition_query=True,
+        )
+
+        if users:
+            user = list(users)
+            return user[0]
         else:
             return None
 
 
-class Task(db.Model):
+class Task:
     class STATUS:
-        COMPLETED = 'COMPLETED'
-        IN_PROGRESS = 'IN_PROGRESS'
-
-    id = db.Column(db.Integer(), primary_key=True)
-    date = db.Column(db.DateTime())
-    task = db.Column(db.String(255))
-    user_id = db.Column(db.String(255))
-    status = db.Column(db.String(255))
-    
+        COMPLETED = "COMPLETED"
+        IN_PROGRESS = "IN_PROGRESS"
+
     def __init__(self, task, user_id, status):
-        self.date = datetime.utcnow().date()
+        self.id = str(randint(_MIN, _MAX))
```

Comment:
I would think Cosmos has a way to generate random identifiers for documents - what happens if you just *don't* assign an id? I would think it would auto generate one.

Reply:
Also, randint doesn't guarantee unique across all items, so please switch over to using Guid for ids.  If you don't provide an "id" field, then Cosmos should auto create an id field and insert a guid.

Reply:
You have to add an id to each item you insert into the container according to the docs here. 
https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/cosmos/azure-cosmos#insert-data

I'll try to use Guid.